File: Python/PRAK603-2210817310017-Rohid Akmal.py
b1, b2 = map(int, input().split())
if b1 == b2 :
    angka1 = list(map(int, input().split()))
    angka2 = list(map(int, input().split()))
    baris1 = []
    baris2 = []
    a = 0
    baris1.append(angka1[a : a + b1])
    baris2.append(angka2[a : a + b2])
    for i in range(0, b1) :
        print(baris1[0][i] * baris2[0][i], end=' ')
else :
    print("Jumlah tidak sama")

File: C/PRAK603-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    int b1, b2, i, in, hsl;
    scanf("%d %d", &b1, &b2);
    int baris1[b1], baris2[b2];
    if (b1 == b2){
        for (i = 0; i < b1; i++){
            scanf("%d", &in);
            baris1[i] = in;}
        for (i = 0; i < b2; i++){
            scanf("%d", &in);
            baris2[i] = in;}
        for (i = 0; i < b1; i++){
            hsl = baris1[i] * baris2[i];
            printf("%d ", hsl);}}
    else {
        printf("Jumlah tidak sama");}
}